feature/onboarding/src/main/kotlin/net/thunderbird/feature/onboarding/OnboardingContent.kt

```diff
@@ -0,0 +1,165 @@
+                    )
+                },
+                verticalArrangement = Arrangement.spacedBy(MainTheme.sizes.large, Alignment.CenterVertically),
+            ) {
+                item {
+                    WelcomeLogo(
+                        modifier = Modifier.fillMaxWidth(),
+                    )
+                }
+                item {
+                    WelcomeTitle(
+                        modifier = Modifier.fillMaxWidth(),
+                    )
+                }
+                item {
+                    WelcomeMessage(
+                        modifier = Modifier.fillMaxWidth(),
+                    )
+                }
+            }
+        }
+    }
+}
+
+@Composable
+private fun WelcomeLogo(
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = Modifier.then(modifier),
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+        Box(
+            modifier = Modifier
+                .size(MainTheme.sizes.huge)
+                .clip(shape = RoundedCornerShape(percent = 50))
+                .background(color = MainTheme.colors.surface),
+        ) {
+            Image(
+                painter = painterResource(id = MainTheme.images.logo),
+                contentDescription = stringResource(id = R.string.onboarding_welcome_logo_content_description),
+                modifier = Modifier.size(MainTheme.sizes.huge),
+            )
+        }
+    }
+}
+
+@Composable
+private fun WelcomeTitle(
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = modifier,
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+        TextHeadline2(
+            text = stringResource(id = R.string.onboarding_welcome_title),
+        )
+    }
+}
+
+@Composable
+private fun WelcomeMessage(
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = Modifier
+            .padding(start = MainTheme.spacings.quadruple, end = MainTheme.spacings.quadruple)
+            .then(modifier),
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+        TextBody1(
+            text = stringResource(id = R.string.onboarding_welcome_message),
+        )
+    }
+}
+
+@Composable
+private fun WelcomeFooter(
+    onStartClick: () -> Unit,
+    onImportClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = modifier,
+        verticalArrangement = Arrangement.spacedBy(MainTheme.spacings.default),
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+        Button(
+            text = stringResource(id = R.string.onboarding_welcome_start_button),
+            onClick = onStartClick,
+        )
+        ButtonText(
+            text = stringResource(id = R.string.onboarding_welcome_import_button),
+            onClick = onImportClick,
+        )
+    }
+}
+
+@Composable
+@DevicePreviews
+internal fun OnboardingContentK9Preview() {
+    K9Theme {
+        OnboardingContent({}, {})
```

Comment:
I know it's only for previews. Still, it would be nicer it this used named arguments.

```kotlin
OnboardingContent(onStartClick = {}, onImportClick = {})
```

Reply:
We could enable a detekt rule to enforce named arguments in the project.